### PR TITLE
Pull request from dev to master

### PR DESCRIPTION
**Pull request by qik-in-mate**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    hideSubtitle(card) {
+      card.isShowSubtitle = false;
+    },
+    showSubtitle(card) {
+      card.isShowSubtitle = true;
+    },
+    addCard() {
+      this.count = ++this.count;
+      this.cards.push({
+        id: this.count++,
```
Comment (carrot-e) [inline index.js]: You don't need to increment here (as you have already incremented `this.count` in line 43.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      this.cards.splice(index, 1);
+    },
+    hideSubtitle(card) {
+      card.isShowSubtitle = false;
+    },
+    showSubtitle(card) {
+      card.isShowSubtitle = true;
+    },
+    addCard() {
+      this.count = ++this.count;
```
Comment (carrot-e) [inline index.js]: You don't need to explicitly overwrite `this.count`.
This:
`++this.count;`
and this:
`this.count++;`
will work too.

Comment (qik-in-mate) [reply index.js]: Count edited, updated.

File: index.js (hunk 1 above)
Comment (qik-in-mate) [reply index.js]: Count edited, updated.

**Pull request merged**